static/src/javascripts/projects/common/modules/identity/sign-in-gate/helper.js

```diff
@@ -256,9 +256,15 @@ export const showGate: ({
                 articleBody.children[0].clientHeight < 125 &&
                 articleBody.children.length > 1
             ) {
-                shadowOverlay.appendChild(
-                    articleBody.children[1].cloneNode(true)
-                );
+                // find the indexes of the articles "p" tag, to include in the sign in gate fade
+                const pIndexes: Array<number> = Array.from(articleBody.children).map((elem, idx) => elem.tagName === "P" ? idx : 0).filter(i => i);
```

Comment:
could any of the p tags have an index of 0?

Reply:
yeah it could, but the first p tag is always inserted anyway before, so the next p tag wont have index 0

though to make it more explicit i've changed it so that it will keep the index 0, and have a check to see if a second p element exists

Reply:
had to use the empty string check to get pass the flow typecheck